src/main/java/org/jabref/gui/entryeditor/SourceTab.java

```diff
@@ -71,7 +73,7 @@ private static String getSourceString(BibEntry entry, BibDatabaseMode type, Late
     private CodeArea createSourceEditor() {
         CodeArea codeArea = new CodeArea();
         codeArea.setWrapText(true);
-        codeArea.lookup(".styled-text-area").setStyle("-fx-font-size: " + Globals.prefs.getFontSizeFX() + "pt;");
+        codeArea.lookup(".styled-text-area").setStyle("-fx-font-size: " + preferences.getFontSizeFX() + "pt;");
```

Comment:
Why not pass the font size directly as int parameter? Would also work for test

Reply:
That would work but there are 2 other accesses to the preferences. Thus 3 new constructor arguments just to get rid of the preference class seems a bit too much for me.